modules/widgets/simple-payments.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Disable direct access/execution to/of the widget code.
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! class_exists( 'Jetpack_Simple_Payments_Widget' ) ) {
+	/**
+	 * Simple Payments Button
+	 *
+	 * Display a Simple Payment Button as a Widget.
+	 */
+	class Jetpack_Simple_Payments_Widget extends WP_Widget {
+		/**
+		 * Constructor.
+		 */
+		function __construct() {
+			$widget = array(
+				'classname' => 'simple-payments',
+				'description' => __( 'Add a Simple Payment Button as a Widget.', 'jetpack' ),
+				'customize_selective_refresh' => true,
+			);
+
+			parent::__construct( 'Jetpack_Simple_Payments_Widget', __( 'Simple Payments', 'jetpack' ), $widget );
```

Comment:
Could you add the `` filter here?

Reply:
Updated on 5d749a37cc859351c0ee28fb728683d1cdd9d930.